src/index.css

```diff
@@ -1,10 +1,10 @@
 body {
-  @import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
-  font-family: 'Rubik', system-ui, -apple-system, sans-serif;
+  @import url("https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap");
+  font-family: "Rubik", system-ui, -apple-system, sans-serif;
 
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
```

Comment:
I think your linter has an opinion on quotes. Maybe we need Prettier or something to standardize styles? Anyway, let's keep it as single quotes.